src/applications/mhv/secure-messaging/components/ManageFolderButtons.jsx

```diff
@@ -65,7 +65,7 @@ const ManageFolderButtons = () => {
 
   const openDelModal = () => {
     dispatch(closeAlert());
-    if (messages?.length > 0) {
+    if (threads.threadList.length > 0) {

```

Comment:
we probably should cover this logic in a unit test as well. Please pardon if you already do, I may have missed